deploy/29_faucet.ts

```diff
@@ -34,14 +35,19 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
 
   const network = networks[hre.networkName];
 
+  const environment: Environment = getEnvironment();
+
   const salt = hre.deploymentSalt;
 
   const holograph = await hre.ethers.getContract('Holograph', deployer);
   const hlgTokenAddress = await holograph.getUtilityToken();
 
   const currentNetworkType: NetworkType = network.type;
 
-  if (currentNetworkType == NetworkType.local || currentNetworkType == NetworkType.testnet) {
+  if (
```

Comment:
are double `==` used purposely instead of `===` in these instances?

Reply:
okay yeah not a big deal right now, but prefer to keep things strict where possible to avoid weird javascript bugs